parser/terror/terror_test.go

```diff
@@ -142,7 +142,7 @@ func TestTraceAndLocation(t *testing.T) {
 			sysStack++
 		}
 	}
-	require.Equalf(t, 11, len(lines)-(2*sysStack), "stack =\n%s", stack)
+	require.Equalf(t, 13, len(lines)-(2*sysStack), "stack =\n%s", stack)
```

Comment:
Could you check the stack of the error stack? I just curious what the changes look like.

Reply:
we use bazel to run tests and bazel will manage Golang environment so that we find the stack path is different from running with ```gotest```.

I have added a comment in the code to describe this problem.